src/extension.ts

```diff
@@ -199,28 +199,51 @@ async function startClients(folders = prunedFolderNames()) {
 	console.log('Starting:', folders);
 	const command = await pathToBinary();
 	const disposables: vscode.Disposable[] = [];
-	for (const folder of folders) {
-		if (!clients.has(folder)) {
-			const commandPrefix = shortUid.seq();
-			const client = newClient(command, folder, commandPrefix);
-			client.onReady().then(() => {
-				reporter.sendTelemetryEvent('startClient');
-			});
-			client.onDidChangeState((event) => {
-				if (event.newState === ClientState.Stopped) {
-					clients.delete(folder);
-					reporter.sendTelemetryEvent('stopClient');
-				}
-			});
-			disposables.push(client.start());
-			clients.set(folder, { commandPrefix, client });
-		} else {
-			console.log(`Client for folder: ${folder} already started`);
+
+	const firstClient = configureClient(command, folders[0])
+	firstClient.client.onReady().then(() => {
+		reporter.sendTelemetryEvent('startClient');
+		const wsSupported = firstClient.client.initializeResult.capabilities.workspace.workspaceFolders.supported
+
+		console.log("Multi-folder support: ", wsSupported)
+		for (var i = 1; i < folders.length; i++) {
```

Comment:
Is there a cleaner/better way of iterating over an array while skipping the first item? Something like `folders[1:]` in Go or Python? 😄

Reply:
In javascript it's `folders.slice(1)`